binding_generator.py

```diff
@@ -964,8 +964,20 @@ def generate_builtin_class_source(builtin_api, size, used_classes, fully_used_cl
             result.append(method_signature + "{")
 
             method_call = "\t"
+            need_additional_right_bracke = False
```

Comment:
There's a type-o here - this variable name should end with a `t`. However, I think it'd be better if this was made to look more like the code for engine classes which use `is_ref` as the variable for this. See https://github.com/godotengine/godot-cpp/blob/master/binding_generator.py#L1585